src/main.rs

```diff
@@ -141,12 +141,22 @@ fn siren() -> anyhow::Result<()> {
 
 fn main() -> anyhow::Result<()> {
     if let Some(arg) = std::env::args().nth(1) {
-        if arg == "--version" {
-            let name = env!("CARGO_PKG_NAME");
-            let version = env!("CARGO_PKG_VERSION");
-            eprintln!("{name}: {version}");
-            std::process::exit(0);
+        let name = env!("CARGO_PKG_NAME");
+        let version = env!("CARGO_PKG_VERSION");
+        match arg.as_str() {
+            "-v" => eprintln!("{name}: {version}"),
+            _ => {
+                eprintln!(
+                    "Usage: {name} [-v] [-h]\n\
```

Comment:
To make this easy, and because when adding another command line switch I would introduce this nonetheless: Do you think I should introduce `clap` yet to make this easier?